src/test/resources/Flipkart.csv

```diff
@@ -0,0 +1,2 @@
+"shoes","1"
```

Comment:
change the name of this file to testname.csv.
Where testname is the name of your test method